Lib/idlelib/colorizer.py

```diff
@@ -132,7 +132,7 @@ def LoadTagDefs(self):
             # when fixing the "hit" tag's visibility, or when the replace dialog is replaced with a
             # non-modal alternative.
             "hit": idleConf.GetHighlight(theme, "hit"),
-            }
+        }
 
```

Comment:
I meant for this dedent to be reverted in #25479.  PEP-8 allows it either way, and I prefer it the way it was.  The blank line before the debug line is not needed and is not used elsewhere.
```suggestion
            }
```